include/featurelib.h

```diff
@@ -0,0 +1,18 @@
+#ifndef FEATUREDETECT_FEATURELIB_H_
+#define FEATUREDETECT_FEATURELIB_H_
+
+#include <vector>
+#include <iostream>
+#include <opencv2/opencv.hpp>
+#include <opencv2/core/core.hpp>
+#include <opencv2/features2d.hpp>
+#include <opencv2/imgproc.hpp>
+
+class featurelib  {
+public:
```

Comment:
`public`/`private` constructs should be indented with one space, according to [our codestyle](https://google.github.io/styleguide/cppguide.html#Class_Format). You can setup CLion to also do it for you.

Reply:
done